contrib/tx_commands.sh

```diff
@@ -1,3 +1,8 @@
+if [ -z "$TRANSIFEX_USER" ]; then
```

Comment:
Is there a way to identify a build from PRs instead? I'm afraid checking for `$TRANSIFEX_USER` may hide an misconfiguration in case we forget to set it up in CircleCI, or any other CI tool we may wan to use in the feature.